src/coreclr/runtime.proj

```diff
@@ -1,7 +1,18 @@
 <Project Sdk="Microsoft.Build.NoTargets">
 
+  <PropertyGroup>
+    <NativeBuildPartitionPropertiesToRemove>ClrFullNativeBuild;ClrRuntimeSubset;ClrJitSubset;ClrPalTestsSubset;ClrAllJitsSubset;ClrILToolsSubset</NativeBuildPartitionPropertiesToRemove>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="runtime-prereqs.proj" GlobalPropertiesToRemove="$(NativeBuildPartitionPropertiesToRemove)" />
```

Comment:
Why do it like this, and not the old pre-reqs way. This one assumes that all targets there are properly incremental. I have no way of just avoiding touching that project anymore as with the old subset route.

Reply:
@ViktorHofer and @agocke have expressed interest in moving towards a world where we use incremental `ProjectReference`s instead of loose coupling between subsets. I figured since the properties here are all incremental, we could test out that process here.